Tools/TestWebKitAPI/Tests/WebKitCocoa/WebPushDaemon.mm

```diff
@@ -29,6 +29,7 @@
 #import "HTTPServer.h"
 #import "PlatformUtilities.h"
 #import "Test.h"
+#import "TestNavigationDelegate.h"
```

Comment:
I'm landing a fix for this soon. https://github.com/WebKit/WebKit/pull/5244

Reply:
Above PR landed in this [commit](https://github.com/WebKit/WebKit/commit/28eb5d639d4fac089bd6a86e2c59560dac26ded5). Can we close this old PR?